docs/user-guide/configuring.md

```diff
@@ -557,13 +557,15 @@ You can tell ESLint to ignore specific files and directories by creating an `.es
 
 When ESLint is run, it looks in the current working directory to find an `.eslintignore` file before determining which files to lint. If this file is found, then those preferences are applied when traversing directories. Only one `.eslintignore` file can be used at a time, so `.eslintignore` files other than the one in the current working directory will not be used.
 
-Globs are matched using [minimatch](https://github.com/isaacs/minimatch), so a number of features are available:
+Globs are matched using [node-ignore](https://github.com/kaelzhang/node-ignore), so a number of features are available:
 
 * Lines beginning with `#` are treated as comments and do not affect ignore patterns.
+* Paths are relative to `.eslintignore` location or the current working directory.
+* Ignore patterns behave according to the `.gitignore` [specification](http://git-scm.com/docs/gitignore)
```

Comment:
Except apparently they do not behave according to the gitignore specification; the gitignore specification requires **s for many of the cases here.